webui/react/src/services/api.ts

```diff
@@ -17,28 +16,17 @@ import { isExperimentTask } from 'utils/task';
 
 import { decodeExperimentList, encodeExperimentState } from './decoder';
 
-const address = serverAddress();
-export const detAuthApi = new DetSwagger.AuthenticationApi(undefined, address);
-export const detExperimentApi = new DetSwagger.ExperimentsApi(undefined, address);
-export const detExperimentsStreamingApi = DetSwagger.ExperimentsApiFetchParamCreator();
+export { isAuthFailure, isLoginFailure, isNotFound } from './utils';
 
-/* Authentication */
-
-/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
-export const isAuthFailure = (e: any): boolean => {
-  return e.response && e.response.status && e.response.status === 401;
-};
+const address = `${window.location.protocol}//${window.location.host}`;
```

Comment:
blocking: Keep as before? 
can we keep the logic coming up with the serverAddress in one location? would also help later when we make that  configurable

Reply:
maybe I should add a note here, but by importing anything from `route/utils` or `utils/route` within `services/api`, it causes a circular dependency.